libs/@local/hql/cst/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(allocator_api, box_into_boxed_slice)]
+
+extern crate alloc;
```

Comment:
Do I see correctly, that we only need `alloc` for the types in `arena::` and we don't use an allocator from here? Great!

Reply:
That is correct! All allocations go through the arena.